tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -107,7 +107,7 @@
       // The endpoint that serves trace viewer app.
       traceViewerBaseUrl: {
         type: String,
-        value: () => getRouter().pluginRoute("profile", "/trace_viewer"),
+        value: "/trace_viewer_index.html",
```

Comment:
The iframed trace viewer app expects a trace data URL to be passed as a parameter.